client/main.go

```diff
@@ -148,3 +105,16 @@ func TinkWorkflowClient() (workflow.WorkflowServiceClient, error) {
 	}
 	return workflow.NewWorkflowServiceClient(conn), nil
 }
+
+// TinkFullClient creates a new full client.
+func TinkFullClient() (FullClient, error) {
+	conn, err := GetConnection()
+	if err != nil {
+		return FullClient{}, err
+	}
+	return FullClient{
```

Comment:
nit: Could `FullClient` just become an interface rather than a struct with three fields that are interfaces?

```go
type FullClient interface {
	template.TemplateServiceClient
	workflow.WorkflowServiceClient
	hardware.HardwareServiceClient
}
```

Reply:
I don't think that'll work as we don't have any concrete struct that would fulfill the FullClient interface. Can you show what you mean?

Reply:
ah nevermind, what you said makes sense